src/test/java/teammates/storage/sqlapi/UsersDbTest.java

```diff
@@ -56,6 +63,17 @@ private Student getTypicalStudent() {
         return new Student(course, "student-name", "valid@teammates.tmt", "comments");
     }
 
+    private Section getTypicalSection() {
+        Course course = new Course("course-id", "course-name", Const.DEFAULT_TIME_ZONE, "institute");
+        return new Section(course, "test-section");
+    }
+
+    private Team getTypicalTeam() {
+        Course course = new Course("course-id", "course-name", Const.DEFAULT_TIME_ZONE, "institute");
+        Section section = new Section(course, "test-section");
+        return new Team(section, "test-team");
+    }
+
```

Comment:
point for discussion, should we move all these getTypicalX into BaseTestCase? this way all the tests can use them, and if necessary, change their attributes using the setting methods we have in each entity

Reply:
Sounds good, moving it can reduce similar redundant methods in the future